app/ai_backend/test_genesis_core.py

```
@@ -0,0 +1,336 @@
+from unittest.mock import patch, MagicMock
+import sys
+import os
+
+# Add the app directory to path for imports
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..', '..'))
+
+try:
+    from app.ai_backend.genesis_core import *
+except ImportError:
+    # If genesis_core doesn't exist, we'll create mock tests that can be adapted
+    pass
```

Ruff (0.11.9)

Use `contextlib.suppress(ImportError)` instead of `try`-`except`-`pass`

(SIM105)